pocs/utils/images/__init__.py

```diff
@@ -7,6 +7,7 @@
 from warnings import warn
 
 from astropy.wcs import WCS
+from astropy.io import fits
 from astropy.io.fits import (getdata, getheader)
```

Comment:
No longer needed, you're not using `getdata` or `getheader`.

Reply:
No problem.

Reply:
No problem.